types/aws-lambda/trigger/sqs.d.ts

```diff
@@ -12,6 +12,7 @@ export interface SQSRecord {
     attributes: SQSRecordAttributes;
     messageAttributes: SQSMessageAttributes;
     md5OfBody: string;
+    MD5OfMessageAttributes?: string;
```

Comment:
This violates the naming convention of every other property in this object - what's going on here?

Reply:
Sorry, wrong copy/paste. It's fixed now.